clang/test/Driver/sycl-dump-device-code.cpp

```diff
@@ -33,3 +33,36 @@
 
 // CHK-FSYCL-DUMP-DEVICE-CODE-CWD: llvm-foreach{{.*}} "--out-dir=.{{(/|\\\\)}}" "--" "{{.*}}llvm-spirv"
 // CHK-FSYCL-DUMP-DEVICE-CODE-WIN-CWD: llvm-foreach{{.*}} "--out-dir=.{{(/|\\\\)}}" "--" "{{.*}}llvm-spirv"
+
```

Comment:
We should have a `sycl-dump-device-code-old-model.cpp` and `sycl-dump-device-code.cpp` testing (separate for old and new) to be consistent with other tests that have both functional behaviors.  It also makes it clear what is being tested based on the test name and allows for easier cleanup when we drop old model.

Reply:
Makes sense.

Reply:
Done.